--- a/interface/input.rb
+++ b/interface/input.rb
@@ -217,7 +217,7 @@
     @@optHash.delete('outFile')
 
     # demonstrate a sample command line
-    cmdline = "geotoad.rb"
+    cmdline = "geotoad"
     hidden_opts = ['queryArg', 'user', 'password', 'usemetric', 'verbose']
     # hide unlimited search
     if @@optHash['limitSearchPages'] == 0
@@ -260,7 +260,7 @@
 
 
   def usage
-    puts "::: SYNTAX: geotoad.rb [options] <search>"
+    puts "::: SYNTAX: geotoad [options] <search>"
     puts ""
     puts " -u <username>          Geocaching.com username, required for coordinates"
     puts " -p <password>          Geocaching.com password, required for coordinates"
@@ -333,12 +333,12 @@
     puts " (+) requires gpsbabel  (=) requires cmconvert  (%) requires iconv in PATH"
     puts ""
     puts "::: EXAMPLES:"
-    puts " geotoad.rb -u helixblue -p password 27502"
+    puts " geotoad -u helixblue -p password 27502"
     puts "   find zipcode 27502 (Apex, NC 27502, USA), search 10 mi around, write gpx"
-    puts " geotoad.rb -u john -p password -c unknown -d 3 -x csv -o NC.cvs -q state 34"
+    puts " geotoad -u john -p password -c unknown -d 3 -x csv -o NC.cvs -q state 34"
     puts "   will find all mystery caches with difficulty >= 3 in all of North Carolina"
     puts "   (Be careful: NC has more than 24k active caches!)"
-    puts " geotoad.rb -u ... -p ... -z -Y -H -c cito -x list -o cito.list -q country 11"
+    puts " geotoad -u ... -p ... -z -Y -H -c cito -x list -o cito.list -q country 11"
     puts "   creates a list (with dates, but no coordinates) of all CITO events in the UK"
     puts " for more examples - and options explanations - see manual page and README"
   end
