--- a/geotoad.rb
+++ b/geotoad.rb
@@ -6,8 +6,10 @@
 
 require 'pathname'
 $BASEDIR = File.dirname(File.realpath(__FILE__))
-$LOAD_PATH << $BASEDIR
-$LOAD_PATH << File.join($BASEDIR, 'lib')
+# For Debian/Ubuntu, modules are below /usr/share/geotoad
+#$LOAD_PATH << $BASEDIR
+#$LOAD_PATH << File.join($BASEDIR, 'lib')
+$LOAD_PATH << '/usr/share/geotoad'
 
 if RUBY_VERSION.gsub('.', '').to_i < 191
   puts "ERROR: Ruby version is #{RUBY_VERSION}. 1.9.1 or higher required."
