--- geotoad.trunk/geotoad.rb	2012-09-25 16:56:07.000000000 +0200
+++ geotoad.trunknew/geotoad.rb	2012-09-25 17:01:43.000000000 +0200
@@ -2,9 +2,11 @@
 #
 # This is the main geotoad binary.
 #
-$LOAD_PATH << File.dirname(__FILE__.gsub(/\\/, '/'))
-$LOAD_PATH << File.dirname(__FILE__.gsub(/\\/, '/')) + '/lib'
-$LOAD_PATH << (File.dirname(__FILE__.gsub(/\\/, '/')) + '/' + '..')
+# For Debian/Ubuntu, modules are below /usr/share/geotoad
+#$LOAD_PATH << File.dirname(__FILE__.gsub(/\\/, '/'))
+#$LOAD_PATH << File.dirname(__FILE__.gsub(/\\/, '/')) + '/lib'
+#$LOAD_PATH << (File.dirname(__FILE__.gsub(/\\/, '/')) + '/' + '..')
+$LOAD_PATH << '/usr/share/geotoad'
 
 $isRuby19 = true
 
