--- geotoad.trunk/interface/input.rb	2012-09-25 13:01:26.000000000 +0200
+++ geotoad.trunknew/interface/input.rb	2012-09-25 17:01:48.000000000 +0200
@@ -181,7 +181,7 @@
 
 
     # demonstrate a sample command line
-    cmdline = "geotoad.rb"
+    cmdline = "geotoad"
     hidden_opts = ['queryArg', 'outDir', 'outFile', 'user', 'password', 'usemetric']
     # hide unlimited search
     if @@optHash['limitSearchPages'] == 0
@@ -229,7 +229,7 @@
 
 
   def usage
-    puts "::: SYNTAX: geotoad.rb [options] <search>"
+    puts "::: SYNTAX: geotoad [options] <search>"
     puts ""
     puts " -u <username>          Geocaching.com username, required for coordinates"
     puts " -p <password>          Geocaching.com password, required for coordinates"
@@ -297,8 +297,8 @@
     puts "    + requires gpsbabel in PATH           = requires cmconvert in PATH"
     puts ""
     puts "::: EXAMPLES:"
-    puts "  geotoad.rb -u helixblue -p password 27502"
-    puts "  geotoad.rb -u john -p password -d 3 -s helixblue -f vcf -o NC.vcf \'North Carolina\'"
+    puts "  geotoad -u helixblue -p password 27502"
+    puts "  geotoad -u john -p password -d 3 -s helixblue -f vcf -o NC.vcf \'North Carolina\'"
   end
 
   def showMenu
