--- geotoad.branch/geotoad.rb	2012-09-11 11:11:30.000000000 +0200
+++ geotoad-3.17.2/geotoad.rb	2012-09-17 12:06:45.000000000 +0200
@@ -2,9 +2,11 @@
 #
 # This is the main geotoad binary.
 #
-$LOAD_PATH << File.dirname(__FILE__.gsub(/\\/, '/'))
-$LOAD_PATH << File.dirname(__FILE__.gsub(/\\/, '/')) + '/lib'
-$LOAD_PATH << (File.dirname(__FILE__.gsub(/\\/, '/')) + '/' + '..')
+# For Debian/Ubuntu, modules are below /usr/share/geotoad
+#$LOAD_PATH << File.dirname(__FILE__.gsub(/\\/, '/'))
+#$LOAD_PATH << File.dirname(__FILE__.gsub(/\\/, '/')) + '/lib'
+#$LOAD_PATH << (File.dirname(__FILE__.gsub(/\\/, '/')) + '/' + '..')
+$LOAD_PATH << '/usr/share/geotoad'
 
 $isRuby19 = false
 
